letstest/scripts/test_sdists.sh

```diff
@@ -20,15 +20,6 @@ TEMP_DIR=$(mktemp -d)
 CONSTRAINTS="$TEMP_DIR/constraints.txt"
 cp tools/requirements.txt "$CONSTRAINTS"
 
-# We pin cryptography to 3.1.1 and pyopenssl to 19.1.0 specifically for CentOS 7 / RHEL 7
```

Comment:
So, we still support running on RHEL 7 via snap. What we don't worry about is being packaged on RHEL 7. We do call out to openssl even from inside the snap, so we probably do need to keep that pinning. I'll run the full test suite though to double check.

Reply:
Alright, I've reverted this change. Yeah, pinning to 3.1.1 isn't a problem for the upgrade goal of this PR, since that's still higher than 2.5.0.